Flaky test "test_format_recipient"

### Steps to Reproduce

```
=================================== FAILURES ===================================
____________________________ test_format_recipient _____________________________

    def test_format_recipient():
        """format_recipient should return the expected string"""
        user = UserFactory.build()
>       assert format_recipient(user) == f"{user.name} <{user.email}>"
E       assert '"Dr. John Th...@example.com>' == 'Dr. John Tho...@example.com>'
E         - Dr. John Thornton <cnxeGtARRUcT@example.com>
E         + "Dr. John Thornton" <cnxeGtARRUcT@example.com>
E         ? +                 +
```

--- a/users/utils_test.py
+++ b/users/utils_test.py
@@ -1,9 +1,28 @@
 """Tests for user utils"""
+import pytest
+
 from users.factories import UserFactory
 from users.utils import format_recipient
 
 
-def test_format_recipient():
+@pytest.mark.parametrize(
+    "name,email,expected",
+    [
+        [
+            "Mr. John Thornton",
+            "jthornton@test.edu",
+            '"Mr. John Thornton" <jthornton@test.edu>',
+        ],
+        ["John Thornton", "jthornton@test.com", "John Thornton <jthornton@test.com>"],
+        ["Joanne O'Brien", "jobrien@test.edu", "Joanne O'Brien <jobrien@test.edu>"],
+        [
+            "Cpl. Joanne O'Brien",
+            "jobrien@test.edu",
+            '"Cpl. Joanne O\'Brien" <jobrien@test.edu>',
+        ],
+    ],
+)
+def test_format_recipient(name, email, expected):
     """format_recipient should return the expected string"""
-    user = UserFactory.build()
-    assert format_recipient(user) == f"{user.name} <{user.email}>"
+    user = UserFactory.build(name=name, email=email)
+    assert format_recipient(user) == expected

